packages/kbn-monaco/src/register_globals.ts

```diff
@@ -37,9 +38,8 @@ registerTheme(ESQL_THEME_ID, buildESQlTheme());
 
 const monacoBundleDir = (window as any).__kbnPublicPath__?.['kbn-monaco'];
 
-// @ts-ignore
```

Comment:
Nice to see those `@ts-ignore` removed. Most of the `@ts-ignore` can be replaced by casting to `unknown` or `any`.

Reply:
it's definitely nice to have it out of the way, in this case it felt more appropriate to rely on the typings monaco exposes, then specifically negate the error caused by passing the unspecified member used for testing since 
it's intentional.